ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -187,7 +187,8 @@ let next_server_mode ~mgmt_enabled =
 
 let mgmt_is_enabled () = Server.current_mode () = Any
 
-let run ~__context ~mgmt_enabled =
+let run ~__context ?mgmt_enabled () =
```

Comment:
Why is this turned into an optional argument? I see it does get explicitly called with true or false every time.